packages/vhd-cli/src/commands/copy.js

```diff
@@ -0,0 +1,93 @@
+import { VhdFile, VhdDirectory } from 'vhd-lib'
+
+import { getHandler } from '@xen-orchestra/fs'
+import getopts from 'getopts'
+import { resolve } from 'path'
+import Disposable from 'promise-toolbox/Disposable'
+
+export default async rawArgs => {
+  const {
+    directory,
+    force,
+    help,
+    _: args,
+  } = getopts(rawArgs, {
+    alias: {
+      directory: 'd',
+      force: 'f',
+      help: 'h',
+    },
+    boolean: ['directory', 'force'],
+    default: {
+      directory: false,
+      force: false,
+      help: false,
+    },
+  })
+  if (args.length < 2 || help) {
+    return `Usage: ${this.command} <source VHD> <destination> --directory --force`
+  }
+  const source = args[0]
+  const dest = args[1]
```

Comment:
If you want, you can use destructuring:
```suggestion
  const [source, dest] = args
```